newscoop/src/Newscoop/NewscoopBundle/Resources/translations/article_type_fields.ar.yml

```diff
@@ -1,4 +1,5 @@
 'Usage at automatic statistics': 'استخدام الإحصائيات الأوتوماتية'
+'Show field in article edit screen': null
```

Comment:
you dont have to add this to each file with translation strings, just simply add it to `article_type_fields.en.yml` and it will be populated automatically by localizer. (anyway it can stay like that, it wont conflict)